assets/js/components/Header.stories.js

```diff
@@ -272,28 +271,6 @@ HeaderViewOnly.parameters = {
 	features: [ 'dashboardSharing' ],
 };
 
-export const HeaderWithModuleRecoveryAlert = Template.bind( {} );
```

Comment:
This story should have been removed when `ModuleRecoveryAlert` was removed from the `Header` component.